psalm.xml

```diff
@@ -1,51 +1,82 @@
 <?xml version="1.0"?>
 <psalm
+    findUnusedPsalmSuppress="true"
+    findUnusedCode="true"
+    findUnusedVariablesAndParams="true"
+    strictBinaryOperands="true"
     totallyTyped="true"
```

Comment:
https://psalm.dev/docs/running_psalm/configuration/#totallytyped
this is deprecated and should prefer `errorLevel="1"`

Reply:
Until `totallyTyped` is supported, I will stick to that: I find `errorLeve="1"` to be super-confusing and counter-intuitive, as you need to go to the docs to find out what it implies.

Until then, sticking with the deprecated (better) syntax :D